css/components/Button/button.css

```diff
@@ -0,0 +1,37 @@
+.btn {
+  background: none;
+  border: none;
+  margin: 0.5rem;
+  padding: 1rem 2rem;
+  transition: background-color 0.3s;
+}
+
+.btn:hover {
+  background: var(--COLOR-SLATE-300);
+}
+
+/** Button Squared **/
+
+.btn-squared {
+  border-radius: 1rem;
+}
+
+/** Filled Buttons **/
+
+.btn-f-error {
+  background: var(--COLOR-RED-500);
+  color: var(--COLOR-WHITE);
+}
```

Comment:
Hey Himanshu use better naming  I would say avoid using alphabets in naming as much as possible Because it will hinder readability use words such as fill which you can use instead.

Reply:
Thanks for pointing this out, I was also thinking the same then thought that the name will become long in length. Like for fill, I will use `btn-fill-error`, and for outline `btn-outline-error`, and for hover `btn-outline-error:hover`.

And as you get it right that it is for `fill`, I thought we can understand. Also, I am using comments in code to separate filled from outlined.

So, what you think? I should change it.

Reply:
I will also suggest the same as @Dev-AkshatGupta said, beacause other people won't understand the meaning of the class without visiting the css files.

Reply:
Okay, I will change them. Thanks for reviewing.